app/src/main/res/values/component_colors.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="text_input_edit_text_cursor_color">@color/primary_text_color</color>
+  <color name="toolbar_layout_background_color">@color/oppia_primary</color>
+  <!--Themes.xml-->
```

Comment:
Nit spacing before and after text
`<!-- Themes.xml -->`

Reply:
Done.